--- workspace/Assets/UniversalRenderPipelineGlobalSettings.asset
+++ isolated-build/Assets/UniversalRenderPipelineGlobalSettings.asset
@@ -68,7 +68,21 @@
       - rid: 5275691211262263332
       - rid: 5275691211262263333
     m_RuntimeSettings:
-      m_List: []
+      m_List:
+      - rid: 5275691211262263300
+      - rid: 5275691211262263301
+      - rid: 5275691211262263304
+      - rid: 5275691211262263307
+      - rid: 5275691211262263310
+      - rid: 5275691211262263313
+      - rid: 5275691211262263314
+      - rid: 5275691211262263315
+      - rid: 5275691211262263316
+      - rid: 5275691211262263321
+      - rid: 5275691211262263325
+      - rid: 5275691211262263328
+      - rid: 5275691211262263329
+      - rid: 5275691211262263330
   m_AssetVersion: 10
   m_ObsoleteDefaultVolumeProfile: {fileID: 0}
   m_RenderingLayerNames:
